src/resources/index.ts

```diff
@@ -28,38 +34,60 @@ export default async function(app: Express, db: Db) {
   });
   
   /* Resource Life Cycle Routes */
-  app.post('/api/v1/resources', async (req, res) => {
-      const { insertedId } = await db.collection('resources').
-        insertOne(req.body);
-      res.status(201);
-      res.send(JSON.stringify({ status: 201, id: insertedId }));
-  });
-  
-  app.put('/api/v1/resources/:id', async (req, res) => {
-      const id = req.params.id;
 
-      // create an update doc for MongoDB
-      const updateDoc = {
-        $set: {...req.body}
-      }
+   /**
+   * Dummy route because GUI wants a content upload URL, even when we're just copying a YouTube link
+   * TO DO: remove the upload content requirement from the GUI, so that YouTube (or other web) links no longer have to post to an dummy file upload link
+   */
+    app.post('/api/v1/resources/:id/content/:token', authorizeRequest, async (req, res) => {
+      return res.status(200).send({
+        status: 200,
+        id: req.params.id
```

Comment:
```suggestion
        resource: {
          id: req.params.id
        }
```
It doesn't seem like we need the rest of the resource here, even an empty `resource: {}` should work.